scripts/gh-pages-preview.mjs

```diff
@@ -0,0 +1,15 @@
+import { join } from "path";
```

Comment:
Just as a note, this refactoring could be done on a different PR from the docs PR.

Reply:
Noted